guides/source/active_record_migrations.md

```diff
@@ -456,23 +466,65 @@ example, this would make your migration irreversible.
 
 Column modifiers can be applied when creating or changing a column:
 
-* `limit`        Sets the maximum size of the `string/text/binary/integer` fields.
-* `precision`    Defines the precision for the `decimal` fields, representing the
-total number of digits in the number.
-* `scale`        Defines the scale for the `decimal` fields, representing the
-number of digits after the decimal point.
-* `polymorphic`  Adds a `type` column for `belongs_to` associations.
```

Comment:
polymorphic seems to be removed for organization purposes but not added bellow, is that intentional?

Reply:
As this only applies to the references I moved the polymorphic and the foreign_key options documentation to the References section.

Reply:
makes sense! :smile: